### PR TITLE
pytest added for output workflow_id

### PR DESCRIPTION
**Pull request by egenc**
Purpose of this PR is to add tests for the functionCloudos.process_workflow_list in CloudOS.

Test data:
```
tests/test_data/process_job_list_initial_json.json
```
Tests are made for checking if the function workflow_id working as intended

To run the test:
```
python3 -m pytest -s -v
```
Expected Output:
image

### CODE REVIEW

File: tests/test_clos/test_workflow_id.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+"""pytest is added for checking workflow_id"""
```
Comment (dapineyro) [inline tests/test_clos/test_workflow_id.py:1]:
```suggestion
"""pytest is added for checking Job.workflow_id"""
```
No really needed though.

File: tests/test_clos/test_workflow_id.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from tests.functions_for_pytest import load_json_file
+
+INPUT_PROJECT = "tests/test_data/projects.json"
+INPUT_WORKFLOW = "tests/test_data/workflows.json"
+APIKEY = 'vnoiweur89u2ongs'
+CLOUDOS_URL = 'http://cloudos.lifebit.ai'
+WORKSPACE_ID = 'lv89ufc838sdig'
+PROJECT_NAME = "lifebit-testing"
+WORKFLOW_NAME = "nf-core-deepvariant"
+
```
Comment (dapineyro) [inline tests/test_clos/test_workflow_id.py:15]:
```suggestion


```

File: tests/test_clos/test_workflow_id.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            headers=header,
+            match=[matchers.query_param_matcher(params)],
+            status=201)
+    # start cloudOS service
+    job = Job(apikey=APIKEY,
+        cloudos_url=CLOUDOS_URL,
+        workspace_id=WORKSPACE_ID,
+        cromwell_token=None,
+        project_name=PROJECT_NAME,
+        workflow_name=WORKFLOW_NAME)
```
Comment (dapineyro) [inline tests/test_clos/test_workflow_id.py:52]:
```suggestion
    job = Job(apikey=APIKEY,
              cloudos_url=CLOUDOS_URL,
              workspace_id=WORKSPACE_ID,
              cromwell_token=None,
              project_name=PROJECT_NAME,
              workflow_name=WORKFLOW_NAME)
```

**Review by dapineyro — CHANGES_REQUESTED** [review]
Thanks @egenc ! I left some minor comments about the style. In addition, as the other functions for `Job` objects, please, move the file to `tests/test_jobs`.

**Comment by egenc** [comment]
thanks @dapineyro ! please check the changes.

**Review by dapineyro — APPROVED** [review]
LGTM! :+1:

**Pull request merged**